### ISSUE TITLE
NibeUplink soll stillgelegt werden

### ISSUE THREAD

**Issue by fritzke316** [sebilm/ioBroker.nibeuplink#303]
Die Verbindung zu NibeUplink soll am 30.August unterbrochen werden, wodurch der NibeUplink Adapter wohl auch nict mehr funktionieren wird :(
siehe E-Mail von myuplink.

![Screenshot 2024-06-18 182815](https://github.com/sebilm/ioBroker.nibeuplink/assets/77584528/1caf2769-e400-4577-a121-bae80c6e3c36)

**Comment by mcm1957** [comment]
@mcm1957 zur evidenz

**Comment by sebilm** [comment]
@fritzke316 Danke für den Hinweis. Dass nibeuplink abgeschaltet werden soll, wusste ich schon, deshalb hab ich ja auch den myuplink-Adapter gemacht. Dass es ein konkretes Datum gibt, wusste ich noch nicht. Ich selbst hab erst letzte Woche die Mail bekommen, dass ich wechseln kann. Da stand aber kein Datum drin. Ich beobachte aber auch schon eine Weile, dass immer mehr Leute den myuplink-Adapter nutzen und immer weniger nibeuplink, wobei der immer noch viel mehr Nutzer hat. Mal schauen, ob die wirklich am 30.7. abschalten.
Wenn es so weit ist, werde ich den Adapter entfernen.

**Comment by sebilm** [comment]
Bisher scheint es noch zu laufen, auch wenn deren Webseite behauptet, dass es zum 1.9.2024 abgeschaltet wird. Es gibt auch immer noch mindestens 240 Leute, die den Adapter verwenden :-)

**Comment by mcm1957** [comment]
@sebilm 
Funktioniert der Adapter noch ?

**Comment by sebilm** [comment]
@mcm1957 Ja, erstaunlicherweise geht es noch. Ich denke, Nibe traut sich nicht, das abzuschalten, weil es noch viele nutzen - also nicht nur ioBroker ;-)

**Comment by mcm1957** [comment]
ok dann warten wir mal ab ...

**Comment by sebilm** [comment]
Es wurde scheinbar nun wirklich stillgelegt - die Antwort der API ist da recht eindeutig. Ich werde den Adapter aus den Repos nehmen lasse (Pull request).
